src/types/common.d.ts

```diff
@@ -0,0 +1,21 @@
+export interface NoteAccidental {
```

Comment:
Okay, so `music.js` is sparingly used in the codebase -- do these types belong here? I'd prefer that we only keep global types in `common.d.ts`.

Reply:
@0xfe would it be right to say? Types ony used in music.ts go in music.d.ts Types used else where go in common.d.ts